Snippets/(thisdir) path to current script.sublime-snippet

```diff
@@ -0,0 +1,6 @@
+<!-- See http://www.sublimetext.com/docs/snippets for more information -->
+<snippet>
+    <content><![CDATA[\$Script:thisDir = Split-Path \$MyInvocation.MyCommand.Path -Parent]]></content>
```

Comment:
I completely agree with @vors. I think we should remove this snippet in favor of using ```$PSScriptRoot``` and ```$PSCommandPath```.